UnityProject/Assets/Scripts/Items/Food/LeanDrink.cs

```diff
@@ -0,0 +1,26 @@
+﻿using HealthV2;
+using UnityEngine;
+
+namespace Items.Food
+{
+	public class LeanDrink : DrinkableContainer
```

Comment:
this should probably be a health effect from the reagents system at some point but I'm fine approving as is for now